qadence/execution.py

```diff
@@ -137,7 +137,7 @@ def _(
     state: Union[Tensor, None] = None,
     n_shots: int = 100,
     backend: BackendName = BackendName.PYQTORCH,
-    error: Union[Errors, None] = None,
+    error: Union[Noise, None] = None,
```

Comment:
Why not using the pipe union ?

Reply:
I was receiving complaints when running the CI, but maybe it was something else at the time?